### PR TITLE
Add cherry-pick bot for the pulsar

### PR DESCRIPTION
**Pull request by zymap**
**Motivation**

We want to make the pulsar can cherry-pick automatically. This action will make a closed PR can cherry-pick by the command /pulsarbot cherry-pick to branch-X.Y.

When a PR is merged, the bot will say,

Hey. If you want to cherry-pick this pr to a target branch, please comments '/pulsarbot cherry-pick to branch-X.Y'.
Then you can comment /pulsarbot cherry-pick to branch-X.Y, the bot will open a new PR for cherry-picking the closed PR.

If we open a new PR to do the cherry-pick, we can also use the CI to check the branch.

### CODE REVIEW

File: .github/workflows/ci-pulsarbot.yaml [hunk 1]
```diff
@@ -41,6 +41,28 @@ jobs:
...
         uses: apache/pulsar-test-infra/pulsarbot@master
+
+      - name: Cherry-pick prompt
+        if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
+        uses: apache/pulsar-test-infra/cherry-pick@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.PULSARBOT_TOKEN }}
+        with:
+          type: prompt-comment
+          github_user: codelipenghui
```
Comment (zymap) [inline .github/workflows/ci-pulsarbot.yaml:56]: Just use a github user who has permission to create a branch on the apache/pulsar repo. We can consider creating a bot user to do this.

**Review by codelipenghui — APPROVED** [review]

File: .github/workflows/ci-pulsarbot.yaml (hunk 1 above)
Comment (codelipenghui) [reply .github/workflows/ci-pulsarbot.yaml:56]: Ok, we can update it later

**Review by wolfstudy — APPROVED** [review]
LGTM+1

File: .github/workflows/ci-pulsarbot.yaml (hunk 1 above)
Comment (sijie) [reply .github/workflows/ci-pulsarbot.yaml:56]: @zymap Can you leave a comment in the file?

Comment (zymap) [reply .github/workflows/ci-pulsarbot.yaml:58]: ok.

**Review by sijie — APPROVED** [review]

**Pull request merged**